### PR TITLE
Update UX4_ValidityReliability.ipynb

### PR DESCRIPTION
**Pull request by kristinnbragi**

### CODE REVIEW

File: UX4_ValidityReliability.ipynb [hunk 1]
```diff
@@ -198,14 +198,14 @@
...
     "\n",
-    "Depending on the type of data (continous, discrete, distribution etc) there are different ways to calculate the confidence intervals. For continuous data we can relate back to the standard error of the mean, which is the standard deviation for the distribution of sample means. Since this is normally distributed, we know that 95% of values lie within  $\\pm 1.96 \\sigma_{\\bar{x}}$  on each side. So a rough estimate of the 95% confidence interval is four times the standard error around the sample mean."
+    "Depending on the type of data (continous, discrete, distribution, etc.) there are different ways to calculate the confidence intervals. For continuous data we can relate back to the standard error of the mean, which is the standard deviation for the distribution of sample means. Since this is normally distributed, we know that 95% of values lie within  $\\pm 1.96 \\sigma_{\\bar{x}}$  on each side. So a rough estimate of the 95% confidence interval is four times the standard error around the sample mean."
```
Comment (kristinnbragi) [inline UX4_ValidityReliability.ipynb:208]: I felt that this paragraph was hard to read. Perhaps it is due to the following: a rough estimate of the 95% confidence interval is four times the standard error around the sample mean.
This sounds like confidence interval is 4 standard deviations. While I know what that means, the students might get confused.

File: UX4_ValidityReliability.ipynb [hunk 2]
```diff
@@ -307,15 +307,15 @@
...
-    "Here is the probability density of the t-distribution (red) for n=2 (1 degree of freedom) compared to the normal distribution (red):\n",
+    "Here is the probability density of the t-distribution (red) for n = 2 (1 degree of freedom) compared to the normal distribution (red):\n",
     "![](https://upload.wikimedia.org/wikipedia/commons/c/cf/T_distribution_1df_enhanced.svg \"Title\")\n",
-    "IkamusumeFan, CC BY-SA 3.0 <https://creativecommons.org/licenses/by-sa/3.0>, via Wikimedia Commons\n",
     "\n",
+    "IkamusumeFan, CC BY-SA 3.0 <https://creativecommons.org/licenses/by-sa/3.0>, via Wikimedia Commons\n",
     "\n",
-    "Therefore, if we want to make a more accurate estimation of confidence intervals we use the value of the t-distribution (instead of the normal) for the critical value (or $\\alpha$level) that we are interested in. \n",
+    "Therefore, if we want to make a more accurate estimation of confidence intervals we use the value of the t-distribution (instead of the normal) for the critical value (or $\\alpha$ level) that we are interested in. \n",
     "$$ \\bar{x}\\pm t_{(1-\\frac{\\alpha}{2})}\\frac{s}{\\sqrt{n}}$$\n",
```
Comment (kristinnbragi) [inline UX4_ValidityReliability.ipynb:316]: This t-distribution formula is not very well explained, and the students might get confused with it. Maybe we should explain it better or just leave it out? I think the graph where we compare normal distribution to t-distribution is clear

File: UX4_ValidityReliability.ipynb [hunk 3]
```diff
@@ -372,14 +373,14 @@
     }
    ],
    "source": [
-    "#find t-statsistic for critical value of upper tail where p=0.025\n",
+    "# Find t-statsistic for critical value of upper tail where p = 0.025\n",
```
Comment (kristinnbragi) [inline UX4_ValidityReliability.ipynb:376]: "p" has never been introduced. Shouldn't this be "(alpha level) of 0.05" (same as previous code block)?

File: UX4_ValidityReliability.ipynb [hunk 4]
```diff
@@ -629,9 +635,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Task time and reaction time data often show positively skewed distributions, which means that arithmetic means are less suitable. As Sauro & Lewis argue, the geometric mean makes a better estimate (in particular for small sample sizes and the confidence interval calculations should be based on the t-value *for log transformed* data.\n",
+    "Task time and reaction time data often show positively skewed distributions, which means that arithmetic means are less suitable. As Sauro & Lewis argue, the geometric mean makes a better estimate (in particular for small sample sizes) and the confidence interval calculations should be based on the t-value for *log transformed* data.\n",
```
Comment (kristinnbragi) [inline UX4_ValidityReliability.ipynb]: Add the source to Sauro & Lewis. I suggest making the "Sauro & Lewis" clickable by doing [Sauro & Lewis](LINK)

File: UX4_ValidityReliability.ipynb [hunk 5]
```diff
@@ -663,21 +669,21 @@
     }
    ],
    "source": [
-    "# logarithmically transformed values\n",
```
Comment (kristinnbragi) [inline UX4_ValidityReliability.ipynb:666]: I don't feel like Logarithmically transformed values has been well introduced in the notebooks, and it is hard to see the difference in the graph. Maybe add a sentence that highlights what it means?

File: UX4_ValidityReliability.ipynb (hunk 4 above)
Comment (sofiadahl) [reply UX4_ValidityReliability.ipynb]: Good idea, can you incorporate that before I merge?

File: UX4_ValidityReliability.ipynb (hunk 5 above)
Comment (sofiadahl) [reply UX4_ValidityReliability.ipynb:666]: True. Actually, this is the first version and I anticipate I'll have to tweak things for next year. But I don't have much time each week for this and it does take time to organize the material in a good way... For now, I'll handle this in class.

File: UX4_ValidityReliability.ipynb (hunk 4 above)
Comment (kristinnbragi) [reply UX4_ValidityReliability.ipynb]: It has now been added

**Pull request merged**